addOns/spider/src/main/resources/org/zaproxy/addon/spider/resources/Messages.properties

```diff
@@ -0,0 +1,2 @@
+spider.addon.name = Spider Add-on
```

Comment:
`Add-on` → `Extension`

Reply:
Should I have left it all in/with the longer file system path?